bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java

```diff
@@ -78,6 +78,9 @@
      */
     private @Nullable ScheduledFuture<?> reconciliationFuture;
 
+    // List of futures used for command retries
+    private List<ScheduledFuture<?>> retryFutures = new ArrayList<ScheduledFuture<?>>();
```

Comment:
Just need to make the collection thread safe.
```suggestion
    private Collection<ScheduledFuture<?>> retryFutures = new ConcurrentLinkedQueue<ScheduledFuture<?>>();
```